### ISSUE TITLE
Implement Citations Section

### ISSUE THREAD

**Issue by mpitblado** [gbif/hp-beaty-biodiversity-museum#9]
Following the removal of the previous custom citation page in 0ad1918968f0e9b1f0e332d0a6937c2900bbfbff, implement the citation section [as shown](https://hp-theme.gbif-staging.org/literature/search) in the base theme https://hp-theme.gbif-staging.org/literature/search, scoped to literature that references the BBM

**Comment by MortenHofft** [comment]
I sometimes wonder if this place in particular needs an explanation. What is your impression? And do you have suggestions for how it should look/work?

**Comment by mpitblado** [comment]
**My two cents (all just suggestions)**

I asked some of the curators here at the museum what matters most to them in terms of citations, and two main themes came up

## Making it easy for users to understand how to properly cite things
One of the big problems that institutions, collections, (and probably GBIF as well) face is that great systems are set up to track when citations are made, however the researcher publishing the paper doesn't properly link the paper to all of the proper systems. This widget (below) has already been developed, and is probably something that could be nicely included in the literature/citations page for an institution.

![citation-explanation-widget](https://github.com/gbif/hp-beaty-biodiversity-museum/assets/116753126/d5c9b1cf-e6fb-4db5-a0b2-efc1f66b0cec)

There could be a filter box placed (already in place for occurrences, dataset searches etc) above this widget that changes the citation content. This means that visually, the citations widget can exist just once, limiting screen real estate, but be changed depending on what the user would like to cite from within the scope of the portal. Importantly, this scales no matter the size and scope of the portal, all that would need to happen is to have filters set up in `_config` depending on which options should be made available to the user, and those options are available within the filter.

![filter-functionality-example](https://github.com/gbif/hp-beaty-biodiversity-museum/assets/116753126/eb075931-fb11-4b28-b48d-c17f351790ab)

While these citation widgets already exist on the bottom of dataset pages, they are easy to miss, and there are additional cases in which someone uses the portal to look up something (say, a university student looks up the length of a bird wing for a paper directly on the portal, and doesn't download anything, they should know how to cite the collection they looked that information up in) and credit the collection and GBIF appropriately. Here at the museum, we could then tell instructors/students that this is the one place they have to look whenever they are accessing information through the portal.

## Literature, Citations and Metrics
The current table for literature linked to the portal is fantastic, and should be kept, but it doesn't need to take the whole screen (similar to how the gbif citation widget can just take up a smaller space on the screen, but in this case the user can also search more easily). Instead, graphs (such as those that are used in the dashboards for collections/institutions) should have the same functionality for citations over time, the most popular journals that the citations took place in, citations by collection/prep type etc. Graphs convince university/government leadership that the GBIF hosted portal, and the institution it is attached to is working, using a metric that matters to them. There is already a citations aggregate count on the top right of the [dataset layout](https://hp-theme.gbif-staging.org/dataset/4fa7b334-ce0d-4e88-aaae-2e0c138d049e), so it may just be scope changes and then loading that data into graphs. Something akin to the metrics section shown below

![image](https://github.com/gbif/hp-beaty-biodiversity-museum/assets/116753126/a6063368-02a5-47dc-b16c-4da2a2f373ae)


## Overall layout
![image](https://github.com/gbif/hp-beaty-biodiversity-museum/assets/116753126/4b081265-9526-4526-9c2f-3185abe19f6f)

**Comment by MortenHofft** [comment]
Thank you for compiling this @mpitblado 

## Making it easier to cite
I will give this some thought and ask others for ideas. We currently do not have the option to generate a citation string before a download is triggered. That is the point where we reserve/issue a DOI and store a snapshot of the data for future reference. So there is no citation string before the user asks us to generate one.
Similar for collection citations, the data is dynamic since it is attached from dataset sources, but perhaps we could do something here. Especially if the collection has a 1:1 mapping with a dataset.

For publishers we do something that resembles what you mention with graphs about citations
https://hp-theme.gbif-staging.org/publisher/b542788f-0dc2-4a2b-b652-fceced449591/citations

Would you prefer to see citations as a tab on dataset pages similar to the publisher page?

**Comment by mpitblado** [comment]
@MortenHofft makes sense! Cannot generate the citation until the doi has been reserved. 

I would definitely advocate for the citations being added to datasets in the same way they are for publishers, don't see a downside to including the information under the table in the same way! People here would love it, because reporting is often done at the dataset level in addition to the entire institution. 

Is there a document/webpage that I could look through that goes over the difference between a publisher/institution? I'm fairly confident I understand the difference between a collection/dataset.

**Comment by ManonGros** [comment]
Hi @mpitblado This is a question we got several times now so I attempted writing an FAQ on the GRSciColl website. You can check it here: https://github.com/gbif/hp-grscicoll/commit/2a47efeb0849f3b3ef95cabaadfebbdd021e9f0f.
Does this seem sensible to you?

**Comment by mpitblado** [comment]
Hi @ManonGros yes that makes sense to me, thanks for writing it up! Heads up there is a typo in line 49 of that commit, `However, they are some differences:` should be `However, there are some differences:`.

**Comment by ManonGros** [comment]
Thanks @mpitblado, I corrected the typo
